src/containers/reports/ReportsReducer.js

```diff
@@ -3,26 +3,23 @@
  */
 
 import { List, Map, fromJS } from 'immutable';
+import { RequestStates } from 'redux-reqseq';
 
 import {
   deleteReport,
   getReport,
-  getReports,
   updateReport,
+  getReportsByDateRange,
 } from './ReportsActions';
 import { CLEAR_CRISIS_TEMPLATE } from '../crisis/CrisisActionFactory';
-import RequestStates from '../../utils/constants/RequestStates';
 
 const INITIAL_STATE :Map<*, *> = fromJS({
-  actions: {
-    getReports: { error: Map() },
-  },
-  deleteState: RequestStates.PRE_REQUEST,
-  fetchState: RequestStates.PRE_REQUEST,
+  deleteState: RequestStates.STANDBY,
```

Comment:
I think reducing this even further to the following would achieve the same outcome
```
const INITIAL_STATE :Map<*, *> = fromJS({
  [GET_REPORTS_BY_DATE_RANGE]: RequestStates.STANDBY
});
```

since it seems unlikely that a GET_ saga will need to stratify multiple `RequestStates`